src/journal-remote/journal-remote.c

```diff
@@ -243,13 +243,13 @@ int journal_remote_add_source(RemoteServer *s, int fd, char* name, bool own_name
                 r = sd_event_add_defer(s->events, &source->buffer_event,
                                        dispatch_raw_source_until_block, source);
                 if (r == 0)
-                        sd_event_source_set_enabled(source->buffer_event, SD_EVENT_OFF);
+                        (void) sd_event_source_set_enabled(source->buffer_event, SD_EVENT_OFF);
```

Comment:
i'd catch the error here and handle it below.

Reply:
ok, done